src/app/scheduler/scheduled-process/scheduled-process.component.html

```diff
@@ -59,6 +59,10 @@
                 <i class="fa fa-minus-circle" aria-hidden="true"></i>
               </div>
             </td>
+            <td>
+              <button *ngIf='schedule.enabled' class="button is-text" (click)="disableSchedule(schedule.id)">disable</button>
```

Comment:
can't we use `toggle button` instead of these two separate buttons for enable and disable schedule? IMHO that will look good on GUI. cc/ @praveen-garg

Reply:
or if still we want to use this approach we can handle it with one button on  ternary operator condition check.

Reply:
I think simple is better!  better readability!

re: switch on/ off  let's not add 1 more dependency for looks!